mindsdb/__main__.py

```diff
@@ -118,6 +119,11 @@ def close_api_gracefully(apis):
         } for api in api_arr
     }
 
+    if not is_cloud:
+        print('running migrations:')
+        p = subprocess.Popen(["python3 mindsdb/run_migrations.py"], shell=True)
```

Comment:
1. you get access to DB before modifying it
2. this will work only if current work dir = mindsdb package dir.

Reply:
Thanks for the notes. Fixed